arm/Microsoft.Network/firewallPolicies/deploy.bicep

```
@@ -153,6 +153,8 @@ resource firewallPolicy 'Microsoft.Network/firewallPolicies@2021-05-01' = {
   }
 }
 
+// serial deployment is mandatory, otherwise it will fail
```

I'd suggest to elaborate more on the reasons why the deployment would fail otherwise. e.g. (please check if it's what you meant in the description)
```suggestion
// Serial deployment is mandatory for multiple rule collection groups to avoid concurrent access to the base policy. 
```

---

CHanged it

---

But I see you changed it completely :D

---

@MrMCake can we resolve the issue?